src/tools/rustfmt/src/types.rs

```diff
@@ -729,8 +729,9 @@ impl Rewrite for ast::Ty {
 
                 rewrite_unary_prefix(context, prefix, &*mt.ty, shape)
             }
-            ast::TyKind::Ref(ref lifetime, _pinned, ref mt) => {
-                // FIXME: format pinnedness
+            ast::TyKind::Ref(ref lifetime, ref mt)
+            | ast::TyKind::PinnedRef(ref lifetime, ref mt) => {
+                // FIXME(pin_ergonomics): correctly format pinned reference syntax
```

Comment:
Why not just fix it rather than suppress it?

Reply:
The nightly style policy states that you can implement formatting, for the record.

Reply:
I was just being lazy. I added the basic support now, although I'm sure it will need more fine tuning to break lines in the right place and all that.